app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -186,79 +238,30 @@
 }
 
 .gem-c-feedback__option-list {
+  display: inline-block;
   list-style-type: none;
-  margin: 0;
   padding: 0;
 }
 
 
 .js-enabled {
-  .gem-c-feedback__js-prompt-questions {
-    @include govuk-media-query($until: tablet) {
-      display: grid;
-    }
-  }
-
-  .gem-c-feedback__prompt-question {
-    @include govuk-media-query($until: tablet) {
-      grid-area: 1 / 1;
-    }
-  }
-
   .gem-c-feedback__option-list {
     list-style-type: none;
     margin: 0;
     padding: 0;
-
-    @include govuk-media-query($until: tablet) {
-      grid-area: 1 / 1 / 2 / 2;
-      display: grid;
-      grid-template-columns: 140px 1fr;
-      // older grid spec
-      grid-row-gap: govuk-spacing(3);
-      // newer grid spec
-      row-gap: govuk-spacing(3); // sass-lint:disable-line no-misspelled-properties
-    }
+    margin-right: govuk-spacing(2);
   }
 
   .gem-c-feedback__option-list-item {
+    position: relative;
+    display: inline-block;
```

Comment:
It's quite hard to follow the inheritance tree through all of this (due to the current duplication of the CSS, not this PR) but I think there's some simplification to be had here. The heading `gem-c-feedback__prompt-question` is also floated and has margin to position it, but I think if all of the floating was removed and everything set to inline-block the required layout can be achieved with fewer lines of code.